puppet/modules/quickstack/manifests/load_balancer/horizon.pp

```diff
@@ -1,29 +1,69 @@
 class quickstack::load_balancer::horizon (
-  $frontend_pub_host,
-  $frontend_priv_host,
-  $frontend_admin_host,
-  $backend_server_names,
-  $backend_server_addrs,
-  $port = '80',
-  $mode = 'http',
-  $log = 'httplog',
+ $frontend_pub_host,
+ $frontend_priv_host,
+ $frontend_admin_host,
+ $backend_server_names,
+ $backend_server_addrs,
+ $backend_port = '80',
+ $mode = 'http',
+ $option = 'httplog'
```

Comment:
This becomes $log after the new patch, and we want it to be tcplog for consistency.